libs/eo/landing-page/shell/src/lib/header/eo-landing-page-header.component.ts

```diff
@@ -0,0 +1,104 @@
+/**
```

Comment:
suggestion: Delete the `header` folder. No reason to put a single file in a folder.